src/components/special/_toggle.less

```diff
@@ -39,24 +41,67 @@ input[type="checkbox"].toggle-btn {
     top: 0;
     z-index: -1;
   }
+
   &:checked:before {
     transform: @toggleSwitchOnPosition;
   }
+
   &:checked:after {
     background-color: @toggleOnBgColor;
   }
+
+  &.xsmall-btn {
+    width: @toggleWidthSmall;
+    height: @toggleHeightSmall;
+    border-radius: @toggleBorderRadiusSmall;
+
+    &.xsmall-btn:checked:before {
```

Comment:
it's not necessary to repeat `.xsmall-btn` once they're all encapsulated inside `&.xsmall-btn` so please remove that wherever it's repetitive